web/src/pages/dashboard.tsx

```diff
@@ -89,7 +89,7 @@ const DashboardPage = (props: Props): ReactElement => {
   });
 
   const CertificationsAlert = useQueryControlledAlert({
-    queryKey: QUERIES.fromTaxRegistration,
```

Comment:
`<TaskProgressCheckbox />` references `QUERIES.fromTaxRegistration`. I don't see it being used else where. Can we delete the key? Are changer required for `<TaskProgressCheckbox />`?